modules/ingestor/src/service/dataset/mod.rs

```diff
@@ -36,7 +36,7 @@ impl<'g> DatasetLoader<'g> {
         }
     }
 
-    #[instrument(skip(self, buffer), ret)]
+    #[instrument(skip(self, buffer), err(level=tracing::Level::INFO))]
```

Comment:
related to other comment (though in reverse) eg. are we not interested in return values as well ?